### ISSUE TITLE
Should Textfield become "Input", and should Search become "Input type=search"?

### ISSUE THREAD

**Issue by eirikbacker** [digdir/designsystemet#2388]
- Input type="search" is a native input anyway- meaning we can add x to clear and icon automatically
- Textfield is a native input anyway
- Button after search can be added as a suffix with `Input.Adornments` element (as discussed in #2311)

**Comment by Barsnes** [comment]
`Search` already has `type="search"`.
Are you suggesting dropping the `Search` component? It has different variants, which makes it more difficult to just append to our `Textfield`

**Comment by eirikbacker** [comment]
Yes; suggesting to remove the component, and rather make the Search button a suffix for Input - rather show example stories on how to combine a button and input instead of making a custom component :)

**Comment by Barsnes** [comment]
This was mentioned in #2221, and should probably be a part of that issue
https://github.com/digdir/designsystemet/issues/2221#:~:text=Denne%20rendrer%20ein%20native%20%3Cinput%3E%2C%20foresl%C3%A5r%20Input%20som%20navn

**Comment by mimarz** [comment]
Suggest we revisit this after #2311

**Comment by mimarz** [comment]
This was solved as part of #2311 and #2708
